submissions/soukainahn/mycode.cpp

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
```

Comment:
Please keep only file submission and remove everything else :) thank you

Reply:
could you please close the other pull request that i opened ,im not  very familiar with this paltform thank you so much

Reply:
do you mean only keeping the source code file

Reply:
exactly yes :)